core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -66,6 +67,10 @@
                                                                            WebRequest request) {
         log.info(ex.getMessage());
         ExceptionResponse exceptionResponse = new ExceptionResponse(getErrorAttributes(request));
+        String detailedMessage = ex.getConstraintViolations().stream()
+            .map(ConstraintViolation::getMessage)
+            .reduce("", (subTotal, element) -> subTotal + element);
```

Comment:
Using `.reduce()` you concatenate `String`s producing a new `String` each time.
It's better to use `.collect(Collectors.joining(""));` that is designed for such cases.